test/boost/cql_query_test.cc

```diff
@@ -5348,6 +5348,27 @@ SEASTAR_TEST_CASE(test_parallelized_select_counter_type) {
     });
 }
 
+SEASTAR_TEST_CASE(test_single_partition_aggregation) {
```

Comment:
This test, specifically, checks that single partition aggregations are not parallelized. I'd adjust this test case name to include information about what is expected, e.g. `test_single_partition_aggregation_is_not_parallelized`. This gives you much more information about the what the test does when you see the name. When such a test fails and you see its name in the test report, you will immediately know what you broke and might not need to look at the source of the test at all.